matter_server/server/device_controller_api.py

```diff
@@ -0,0 +1,1413 @@
+"""API for controller that Manages Matter devices."""
```

Comment:
Can we make this docstring a bit more clear to explain to first-viewers what thius does and more important whats the diference with the device controller ?

Reply:
Done :+1: